sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.h

```diff
@@ -98,6 +102,8 @@ typedef struct _nib_onl_entry {
      * @note    Only available if @ref GNRC_IPV6_NIB_CONF_ARSM != 0.
      */
     uint8_t l2addr[GNRC_IPV6_NIB_L2ADDR_MAX_LEN];
+    evtimer_msg_event_t reach_timeout;  /**< Event for @ref GNRC_IPV6_NIB_REACH_TIMEOUT */
+    evtimer_msg_event_t delay_timeout;  /**< Event for @ref GNRC_IPV6_NIB_DELAY_TIMEOUT */
```

Comment:
Actually `ROUTE_TIMEOUT` can be removed, currently I assume the routing protocol to maintain the routes itself (see `ft_add` function in the same model). Merging the *event object* for `REACH_TIMEOUT` and `DELAY_TIMEOUT` could however be a possibility (as I did with `SND_MC_NS` and `SND_UC_NS`), let me think about it a bit, because there might be race conditions, we did not consider yet.